recipes/utf8.h/all/conanfile.py

```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+from conans import ConanFile, tools
+
+
+class Utf8HConan(ConanFile):
+    name = "utf8.h"
```

Comment:
is the `.` acceptable for the naming convention?
Is this too generic? https://github.com/search?l=C%2B%2B&p=2&q=utf8&type=Repositories

Reply:
I don't see valid reason why dot shouldn't be allowed in the reference. I'd actually allow any characters, maybe except `/` and `@` which are used as separators.

Reply:
We already have a dot in `approvaltests.cpp` package and `utf8.h` is the _name_ of the library. I also feel a bit strange, but IMHO it is a valid character for a reference and it's been the authors' choice, I think it is better to use it than to rename the library.